python/jupyter_suggestions_rtc/.gitignore

```diff
@@ -0,0 +1,2 @@
+jupyter_suggestions_rtc/labextension
```

Comment:
Wondering if we could hoist that to the top-level `.gitignore` so these entries are easier to find?